crates/router/src/configs/settings.rs

```diff
@@ -120,6 +120,8 @@ pub struct Settings<S: SecretState> {
     pub user_auth_methods: SecretStateContainer<UserAuthMethodSettings, S>,
     pub decision: Option<DecisionConfig>,
     pub locker_based_open_banking_connectors: LockerBasedRecipientConnectorList,
+    pub network_tokenization_supported_card_networks: NetworkTokenizationSupportedCardNetworks,
+    pub network_tokenization_service: SecretStateContainer<NetworkTokenizationService, S>,
```

Comment:
Should we consider keeping this optional, since this need not be specified by everyone running hyperswitch?

1. What happens if we do not include the `network_tokenization_service` section in the TOML file?
2. What happens if we do not specify one or more fields in the `network_tokenization_service` section in the TOML file?

Do we see errors thrown in any of the two cases?

Reply:
> What happens if we do not include the network_tokenization_service section in the TOML file?

this below error will be logged - 
`ERROR router::core::payment_methods::network_tokenization: Error while making tokenization request, error: Failed to save network token in vault`, but will not stop any payment flow.

>What happens if we do not specify one or more fields in the network_tokenization_service section in the TOML file?

We will throw Application Configuration Error